### PR TITLE
Makes date/time format parameterized

### PR DESCRIPTION
**Pull request by rbernardino**
Adds -DateTimeFormat parameter to Enable-Logfile

### CODE REVIEW

File: LogFile.cs [hunk 1]
```diff
@@ -65,7 +71,7 @@ public override void WriteDebug(string message)
                 CheckDirectory();
                 if (message != String.Empty)
                 {
-                    message = String.Format("{0,-29} - [D] {1}", DateTime.UtcNow.ToString(DateTimeFormat), message);
+                    message = String.Format("{0,-18} - [D] {1}", DateTime.UtcNow.ToString(DateTimeFormat), message);
```
Comment (dlwyatt) [inline LogFile.cs]: Why change the formatting here?

File: LogFile.cs [hunk 2]
```diff
@@ -11,21 +11,25 @@ public class LogFile : HostIOSubscriberBase
     {
         #region Fields
 
-        private const string DateTimeFormat = "r";
+        //private string DateTimeFormat = "dd-MMM-yyyy - HH:mm:ss";
```
Comment (dlwyatt) [inline LogFile.cs]: No reason to leave a commented line of code around

File: LogFile.cs (hunk 1 above)
Comment (rbernardino) [inline LogFile.cs]: Hi Dave! I changed it back to its original value. I was just doing some testing back then. Thanks!

**Comment by dlwyatt** [comment]
v1.3.0 with this PR has been published to the PowerShell Gallery, and in a GitHub release.  Thanks!  :)

**Pull request merged**